app/assets/javascripts/discourse/app/components/sidebar/anonymous/categories-section.hbs

```diff
@@ -5,6 +5,7 @@
 
   {{#each this.sectionLinks as |sectionLink|}}
     <Sidebar::SectionLink
+      @linkName={{sectionLink.name}}
```

Comment:
Sneaking in another fix here and is covered by the tests since we target the link via a CSS class which is derived from `@linkName`